ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/admin/package_management_service.proto

```diff
@@ -63,14 +63,16 @@ message PackageDetails {
   string package_id = 1;
 
   // Size of the package in bytes.
+  // The size of the package is given by the size of the ``daml_lf``
+  // ArchivePayload. See further details in ``daml_lf.proto``.
   // Required
   uint64 package_size = 2;
 
   // Indicates since when the package is known to the backing participant.
   // Required
   google.protobuf.Timestamp known_since = 3;
 
-  // Descrioption provided by the backing participant descnribing where
+  // Description provided by the backing participant describing where
```

Comment:
Is this really about the origin of the source (e.g. some company's name), or rather a description of the package (e.g. `IOU model v2.7`)?

Reply:
It is an arbitrary string produced by the participant, you can't set it through the ledger API. AFAIR @mziolekda mentioned this is about where/how the packages were uploaded, e.g., `Uploaded by admin ABC through participant node XYZ on 2019-06-26`.

In the ledger API, it is described as `Description provided by the backing participant describing where it got the package from.`

In the `WriteService`, it is described as `The description of the packages provided by the participant implementation.`

The former description looks more correct, I will update the comment in `WriteService`

Reply:
I see. Thank you.